neqo-transport/src/cc/classic_cc.rs

```diff
@@ -110,6 +111,7 @@ pub struct ClassicCongestionControl<T> {
     acked_bytes: usize,
     ssthresh: usize,
     recovery_start: Option<Instant>,
+    last_non_app_limited: PacketNumber,
```

Comment:
Adding some documentation here about what this is used for and what that means would be good.